src/Query/TaskGroup.ts

```diff
@@ -42,12 +52,12 @@ export class TaskGroup {
     public readonly tasks: Task[];
```

Comment:
Would you know what does the `groups()` on the line 40 refer to?

Reply:
> Would you know what does the `groups()` on the line 40 refer to?

I cannot tell from GitHub which line of code this is referring to...